Common.props

```diff
@@ -3,12 +3,19 @@
   <PropertyGroup>
     <Copyright>Sebastien Ros</Copyright>
     <Authors>Sebastien Ros</Authors>
+
```

Comment:
Extra new line

Reply:
I was trying to create a bit of a grouping between logical things, I can also remove if it bothers.